src/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -228,6 +228,8 @@ public File getSQLDialect(DatabaseType db_type) {
                 if (LOG.isDebugEnabled())
                     LOG.warn(String.format("Failed to find SQL Dialect XML file '%s'", xmlName));
         }
+ 
+        LOG.info("Failed to find SQL Dialect XML file ");
```

Comment:
Since you only added a log statement to this file you should just remove it from the pull request.